clients/benchmarks/client.cpp

```diff
@@ -0,0 +1,1311 @@
+
+    // clang-format off
+    desc.add_options()
+
+        ("sizem,m",
+         value<hipblas_int>(&arg.M)->default_value(128),
+         "Specific matrix size: sizem is only applicable to BLAS-2 & BLAS-3: the number of "
+         "rows or columns in matrix.")
+
+        ("sizen,n",
+         value<hipblas_int>(&arg.N)->default_value(128),
+         "Specific matrix/vector size: BLAS-1: the length of the vector. BLAS-2 & "
+         "BLAS-3: the number of rows or columns in matrix")
+
+        ("sizek,k",
+         value<hipblas_int>(&arg.K)->default_value(128),
+         "Specific matrix size: BLAS-2: the number of sub or super-diagonals of A. BLAS-3: "
+         "the number of columns in A and rows in B.")
+
+        ("kl",
+         value<hipblas_int>(&arg.KL)->default_value(128),
+         "Specific matrix size: kl is only applicable to BLAS-2: The number of sub-diagonals "
+         "of the banded matrix A.")
+
+        ("ku",
+         value<hipblas_int>(&arg.KU)->default_value(128),
+         "Specific matrix size: ku is only applicable to BLAS-2: The number of super-diagonals "
+         "of the banded matrix A.")
+
+        ("lda",
+         value<hipblas_int>(&arg.lda)->default_value(128),
+         "Leading dimension of matrix A, is only applicable to BLAS-2 & BLAS-3.")
+
+        ("ldb",
+         value<hipblas_int>(&arg.ldb)->default_value(128),
+         "Leading dimension of matrix B, is only applicable to BLAS-2 & BLAS-3.")
+
+        ("ldc",
+         value<hipblas_int>(&arg.ldc)->default_value(128),
+         "Leading dimension of matrix C, is only applicable to BLAS-2 & BLAS-3.")
+
+        ("ldd",
+         value<hipblas_int>(&arg.ldd)->default_value(128),
+         "Leading dimension of matrix D, is only applicable to BLAS-EX ")
+
+        ("stride_a",
+         value<hipblas_int>(&arg.stride_a)->default_value(128*128),
+         "Specific stride of strided_batched matrix A, is only applicable to strided batched"
+         "BLAS-2 and BLAS-3: second dimension * leading dimension.")
+
+        ("stride_b",
+         value<hipblas_int>(&arg.stride_b)->default_value(128*128),
+         "Specific stride of strided_batched matrix B, is only applicable to strided batched"
+         "BLAS-2 and BLAS-3: second dimension * leading dimension.")
+
+        ("stride_c",
+         value<hipblas_int>(&arg.stride_c)->default_value(128*128),
+         "Specific stride of strided_batched matrix C, is only applicable to strided batched"
+         "BLAS-2 and BLAS-3: second dimension * leading dimension.")
+
+        ("stride_d",
+         value<hipblas_int>(&arg.stride_d)->default_value(128*128),
+         "Specific stride of strided_batched matrix D, is only applicable to strided batched"
+         "BLAS_EX: second dimension * leading dimension.")
+
+        ("stride_x",
+         value<hipblas_int>(&arg.stride_x)->default_value(128*128),
+         "Specific stride of strided_batched vector x, is only applicable to strided batched"
```

Comment:
The default values of m, n, k is 128, the default value of incx is 1. With these defaults, I think the default_value for stride_x should be 128, same for stride_y

Reply:
Very good will reduce these.  I copied from rocBLAS so those could likely be checked for reduction as well.